src/node_i18n.cc

```diff
@@ -603,9 +603,10 @@ static void ToASCII(const FunctionCallbackInfo<Value>& args) {
 // consideration.
 static int GetColumnWidth(UChar32 codepoint,
                           bool ambiguous_as_full_width = false) {
-  if (!u_isdefined(codepoint) ||
```

Comment:
My terminal (GNOME Terminal) actually displays unassigned characters (in a weird box form) so they are more than 0-width. Indeed, [UAX #11](http://unicode.org/reports/tr11/#Unassigned) specifies that

> Unassigned code points in ranges intended for CJK ideographs are classified as Wide.

while

> All other unassigned code points are by default classified as Neutral.

I removed the `u_isdefined()` check here so that these unassigned characters can use the generic ICU routine below, which does the right thing.